### PR TITLE
Feature/14848 instructions page config

### PR DESCRIPTION
**Pull request by jon-shipley**
Remove hardcoded numbers from the Instructions page and use the actual config vars instead

### CODE REVIEW

File: pupil-spa/src/app/instructions/instructions.component.html [hunk 1]
```diff
@@ -5,8 +5,8 @@ <h1 class="heading-xlarge">Instructions</h1>
 <div>
   <ul class="text list list-bullet">
     <li>This check has <strong>{{count}} questions</strong>.</li>
-    <li>Each question will show for <strong>5 seconds</strong>.</li>
-    <li>There will be a <strong>2-second</strong> pause between each question.</li>
+    <li>Each question will show for <strong>{{questionTime}} seconds</strong>.</li>
+    <li>There will be a <strong>{{loadingTime}}-second</strong> pause between each question.</li>
```
Comment (activemq) [inline pupil-spa/src/app/instructions/instructions.component.html]: Do we need the '-', as now its reads like:
"There will be a 3-second pause between each question."
@jon-shipley

Comment (jon-shipley) [inline pupil-spa/src/app/instructions/instructions.component.html]: @activemq - that's exactly as the copy is intended to read, and indeed did read when it was hardcoded to `2-second pause`

**Comment by jon-shipley** [comment]
@activemq - heads up - I suspect this has broken the build.  Can you review the cucumber tests please?  They are now dynamic and can be either from config or the the database.

File: pupil-spa/src/app/instructions/instructions.component.html (hunk 1 above)
Comment (activemq) [inline pupil-spa/src/app/instructions/instructions.component.html]: 👍

**Comment by activemq** [comment]
35 scenarios (35 passed)
79 steps (79 passed)
1m17.946s

**Pull request merged**